src/_pytest/terminal.py

```diff
@@ -1636,3 +1638,14 @@ def _get_raw_skip_reason(report: TestReport) -> str:
         elif reason == "Skipped":
             reason = ""
         return reason
+
+
+class _CallableBool:
+    def __init__(self, value: bool):
+        self._value = value
+
+    def __bool__(self):
```

Comment:
Do we Want to plan for a evenual deprecation of the non call variant?

Reply:
you mean by issuing a warning when it's used?